cmd/transform-swagger/main.go

```diff
@@ -154,9 +154,14 @@ func addSharedExtension(apiSpec *spec.Swagger, sharedDefs map[string]bool) (*spe
 			continue
 		}
 
-		// The shared definition name gets transformed into a camelcased Go type.
-		// example: hashicorp.cloud.common.PaginationRequest -> HashicorpCloudCommonPaginationRequest
-		genTypeName := strcase.ToCamel(sharedDefName)
+		var genTypeName string
+		if sharedDefName != "google.rpc.Status" {
```

Comment:
It would be nice if we could use some sort of map instead to accommodate `iam`, `api`, and other acronym protos from `rpc` as well. You can see what we're referring to here: https://github.com/googleapis/googleapis/tree/master/google